modules/microshift-installing-from-rpm.adoc

```diff
@@ -31,6 +31,13 @@ $ sudo subscription-manager repos \
 $ sudo dnf install -y microshift
 ----
 
+. Optional: Install greenboot for {product-title} by running the following command:
```

Comment:
I'm not sure there's any benefit to the user of installing this package on a regular RHEL system, rather than an ostree-based system. We should include the package in the blueprint for building an ostree image instead.

Reply:
It's already included in the blueprint. See the change above.
For non-ostree systems, we should probably add a note for the limitations of Greenboot, i.e. no-rollback, etc

Reply:
It sounds like this is covered, then.

Reply:
So are we removing this, or adding the note? I got lost in the comments on the outcome here.